.github/workflows/turbo_hierarchy.yml

```diff
@@ -0,0 +1,58 @@
+name: Turborepo Hierarchy
+
+on:
+  push:
+    branches:
+      - "develop"
+      - "devnet"
+      - "testnet"
+      - "mainnet"
+      - "releases/iota-*-release"
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  diff:
+    runs-on: [ubuntu-latest]
```

Comment:
this is not supposed to run on [self-hosted]?

Reply:
No, it's cheap so we want to offload it to a GH runner

Reply:
ah, okay, thanks